.github/workflows/ci.yml

```diff
@@ -1,6 +1,6 @@
 name: CI
 
-on: [push, pull_request]
+on: [push]
```

Comment:
I think we need pull_requests for PRs from forks

Reply:
so let's just keep it on pull requests? avoid the double actions during a PR

Reply:
```suggestion
on: [pull_request]
```